--- a/vendor/propel/util/DebugPDO.php
+++ b/vendor/propel/util/DebugPDO.php
@@ -327,17 +327,12 @@
 	 * @see        http://php.net/manual/en/pdo.query.php for a description of the possible parameters.
 	 * @return     PDOStatement
 	 */
-	public function query()
+	public function query(...$args)
 	{
 		$debug	= $this->getDebugSnapshot();
-		$args	= func_get_args();
-		if (version_compare(PHP_VERSION, '5.3', '<')) {
-			$return	= call_user_func_array(array($this, 'parent::query'), $args);
-		} else {
-			$return	= call_user_func_array('parent::query', $args);
-		}
-		
-		$sql = $args[0];
+		$return = parent::query(...$args);
+		
+		$sql = array_key_exists('query', $args) ? $args['query'] : $args[0];
 		$this->log($sql, null, __METHOD__, $debug);
 		$this->setLastExecutedQuery($sql);
 		$this->incrementQueryCount();
