--- a/api_v3/lib/KalturaFrontController.php
+++ b/api_v3/lib/KalturaFrontController.php
@@ -73,7 +73,7 @@
 			'masterPartnerId' => kCurrentContext::$master_partner_id,
 			'ks' => kCurrentContext::$ks,
 			'isAdmin' => intval(kCurrentContext::$is_admin_session),
-			'kuserId' => '"' . str_replace('"', '\\"', (kCurrentContext::$uid ? kCurrentContext::$uid : kCurrentContext::$ks_uid)) . '"',
+			'kuserId' => '"' . str_replace('"', '\\"', ((kCurrentContext::$uid ? kCurrentContext::$uid : kCurrentContext::$ks_uid) ?? '')) . '"',
 			'duration' => $duration,
 			'success' => intval($success),
 			'errorCode' => $errorCode,
