--- a/vendor/symfony/util/sfCore.class.php
+++ b/vendor/symfony/util/sfCore.class.php
@@ -181,31 +181,9 @@
     {
       ini_set('unserialize_callback_func', 'spl_autoload_call');
     }
-    else if (!function_exists('__autoload'))
-    {
-      ini_set('unserialize_callback_func', '__autoload');
-
-      function __autoload($class)
-      {
-        foreach (sfCore::getAutoloadCallables() as $callable)
-        {
-          if (call_user_func($callable, $class))
-          {
-            return true;
-          }
-        }
-
-        // unspecified class
-        // do not print an error if the autoload came from class_exists
-        $trace = debug_backtrace();
-        if (count($trace) < 1 || ($trace[1]['function'] != 'class_exists' && $trace[1]['function'] != 'is_a'))
-        {
-          $error = sprintf('Autoloading of class "%s" failed. Try to clear the symfony cache and refresh. [err0003]', $class);
-          $e = new sfAutoloadException($error);
-
-          $e->printStackTrace();
-        }
-      }
+    else
+    {
+      throw new RuntimeException('SPL autoload support is required.');
     }
 
     self::addAutoloadCallable(array('sfCore', 'splAutoload'));
@@ -257,14 +235,9 @@
 
       spl_autoload_register(array('sfCore', 'splSimpleAutoload'));
     }
-    elseif (!function_exists('__autoload'))
-    {
-      ini_set('unserialize_callback_func', '__autoload');
-
-      function __autoload($class)
-      {
-        return sfCore::splSimpleAutoload($class);
-      }
+    else
+    {
+      throw new RuntimeException('SPL autoload support is required.');
     }
   }
 }
