--- a/vendor/symfony/util/sfFinder.class.php
+++ b/vendor/symfony/util/sfFinder.class.php
@@ -114,7 +114,7 @@
    */
   protected function to_regex($str)
   {
-    if ($str{0} == '/' && $str{strlen($str) - 1} == '/')
+    if ($str[0] == '/' && $str[strlen($str) - 1] == '/')
     {
       return $str;
     }
@@ -510,10 +510,10 @@
 
   public static function isPathAbsolute($path)
   {
-    if ($path{0} == '/' || $path{0} == '\\' ||
-        (strlen($path) > 3 && ctype_alpha($path{0}) &&
-         $path{1} == ':' &&
-         ($path{2} == '\\' || $path{2} == '/')
+    if ($path[0] == '/' || $path[0] == '\\' ||
+        (strlen($path) > 3 && ctype_alpha($path[0]) &&
+         $path[1] == ':' &&
+         ($path[2] == '\\' || $path[2] == '/')
         )
        )
     {
