--- a/vendor/symfony/util/sfToolkit.class.php
+++ b/vendor/symfony/util/sfToolkit.class.php
@@ -369,7 +369,7 @@
    */
   public static function replaceConstants($value)
   {
-    return is_string($value) ? preg_replace_callback('/%(.+?)%/', create_function('$v', 'return sfConfig::has(strtolower($v[1])) ? sfConfig::get(strtolower($v[1])) : "%{$v[1]}%";'), $value) : $value;
+    return is_string($value) ? preg_replace_callback('/%(.+?)%/', function ($v) { return sfConfig::has(strtolower($v[1])) ? sfConfig::get(strtolower($v[1])) : "%{$v[1]}%"; }, $value) : $value;
   }
 
   /**
