--- a/vendor/IP2Location/IP2Location.inc.php
+++ b/vendor/IP2Location/IP2Location.inc.php
@@ -80,11 +80,11 @@
    foreach ($f as $f_k => $f_v) {
    $repeater = intval (substr ($f_v, 1));
    if ($repeater == 0) $repeater = 1;
-   if ($f_v{1} == '*')
+   if ($f_v[1] == '*')
    {
        $repeater = count ($ar) - $i;
    }
-   if ($f_v{0} != 'd') { $i += $repeater; continue; }
+   if ($f_v[0] != 'd') { $i += $repeater; continue; }
    $j = $i + $repeater;
    for ($a = $i; $a < $j; ++$a)
    {
