--- a/vendor/symfony/vendor/pake/pakeGetopt.class.php
+++ b/vendor/symfony/vendor/pake/pakeGetopt.class.php
@@ -50,14 +50,14 @@
 
   public function add_option($long_opt, $short_opt, $mode = self::NO_ARGUMENT, $comment = '')
   {
-    if ($long_opt{0} == '-' && $long_opt{1} == '-')
+    if ($long_opt[0] == '-' && $long_opt[1] == '-')
     {
       $long_opt = substr($long_opt, 2);
     }
 
     if ($short_opt)
     {
-      if ($short_opt{0} == '-')
+      if ($short_opt[0] == '-')
       {
         $short_opt = substr($short_opt, 1);
       }
@@ -81,7 +81,7 @@
       $args = $this->read_php_argv();
 
       // we strip command line program
-      if (isset($args[0]) && $args[0]{0} != '-')
+      if (isset($args[0]) && $args[0][0] != '-')
       {
         array_shift($args);
       }
@@ -101,12 +101,12 @@
         break;
       }
 
-      if ($arg{0} != '-' || (strlen($arg) > 1 && $arg{1} == '-' && !$this->long_options))
+      if ($arg[0] != '-' || (strlen($arg) > 1 && $arg[1] == '-' && !$this->long_options))
       {
         $this->arguments = array_merge($this->arguments, array($arg), $this->args);
         break;
       }
-      elseif (strlen($arg) > 1 && $arg{1} == '-')
+      elseif (strlen($arg) > 1 && $arg[1] == '-')
       {
         $this->parse_long_option(substr($arg, 2));
       }
@@ -149,7 +149,7 @@
   {
     for ($i = 0; $i < strlen($arg); $i++)
     {
-      $opt = $arg{$i};
+      $opt = $arg[$i];
       $opt_arg = true;
 
       /* option exists? */
@@ -169,7 +169,7 @@
         else
         {
           // take next element as argument (if it doesn't start with a -)
-          if (count($this->args) && $this->args[0]{0} != '-')
+          if (count($this->args) && $this->args[0][0] != '-')
           {
             $this->options[$this->short_options[$opt]['name']] = array_shift($this->args);
             break;
@@ -189,7 +189,7 @@
         else
         {
           // take next element as argument (if it doesn't start with a -)
-          if (count($this->args) && $this->args[0]{0} != '-')
+          if (count($this->args) && $this->args[0][0] != '-')
           {
             $this->options[$this->short_options[$opt]['name']] = array_shift($this->args);
           }
