--- a/vendor/symfony/vendor/pake/pakeYaml.class.php
+++ b/vendor/symfony/vendor/pake/pakeYaml.class.php
@@ -193,7 +193,7 @@
         } elseif ($this->_inBlock == true && empty($ifchk)) {
           $last =& $this->_allNodes[$this->_lastNode];
           $last->data[key($last->data)] .= "\n";
-        } elseif ($ifchk{0} != '#' && substr($ifchk,0,3) != '---') {
+        } elseif ($ifchk[0] != '#' && substr($ifchk,0,3) != '---') {
           // Create a new node and get its indent
           $node         = new pakeYAMLNode;
           $node->indent = $this->_getIndent($line);
