--- a/vendor/symfony/vendor/phing/util/StringHelper.php
+++ b/vendor/symfony/vendor/phing/util/StringHelper.php
@@ -60,7 +60,7 @@
         $ret=array();
         $len=strlen($str);
         for ($i=0; $i < $len; $i++) {
-            $ret[] = $str{$i};
+            $ret[] = $str[$i];
         }
         return $ret;
     }
@@ -177,7 +177,7 @@
             trigger_error("substring(), Endindex out of bounds must be $startpos<n<".($len-1), E_USER_ERROR);
         }
         if ($startpos === $endpos) {
-            return (string) $string{$startpos};
+            return (string) $string[$startpos];
         } else {
             $len = $endpos-$startpos;
         }
